build/steps-build.yml

```diff
@@ -55,16 +55,31 @@
     logProjectEvents: false
     createLogFile: false
 
-- task: VSTest@2
+- script: dotnet test $(Build.SourcesDirectory)/src/$(SolutionFileName) /p:Configuration=Release_Tests /p:CollectCoverage=true /p:IncludeTestAssembly=true /p:CoverletOutputFormat=cobertura /p:ExcludeByFile="**/*.g.cs" /p:Exclude="[*]*.Tests.*" --logger trx --no-build
   displayName: 'Run tests'
   condition: and(succeeded(), eq(variables['ApplicationConfiguration'], 'Release_Tests'))
+
+- task: PublishTestResults@2
+  displayName: Publish test results
+  condition: succeededOrFailed()
+  inputs:
+    testRunner: VSTest
+    testResultsFiles: "**/*.trx"
+
+- task: reportgenerator@4
```

Comment:
Do we absolutely need that? I think Azure DevOps supports the cobertura.xml file directly

Reply:
No we don't!